test/functional/job/job_spec.lua

```diff
@@ -101,9 +101,9 @@ describe('jobs', function()
     nvim('command', notify_job())
     nvim('command', "let j = jobstart('xxx', 'cat', ['-'])")
     local jobid = nvim('eval', 'j')
-    nvim('eval', 'jobsend(j, "abcdef")')
-    nvim('eval', 'jobstop(j)')
+    nvim('eval', 'jobsend(j, "abcdef\n")')
     eq({'notification', 'j', {{jobid, 'stdout', {'abcdef'}}}}, next_message())
+    nvim('eval', 'jobstop(j)')
```

Comment:
I've been thinking that too, maybe use busted `pending` method and come back for fixing later once jobsend can successfully report failure or success?